configure.ac

```diff
@@ -1,5 +1,13 @@
-AC_PREREQ([2.60])
-AC_INIT([libsecp256k1],[0.1])
+AC_PREREQ([2.69])
```

Comment:
Why is this necessary?

Reply:
This was to do what bitcoin core was doing for version and release management, I think I can improve it though

Reply:
I think we shouldn't increase it to 2.69 without a good reason. I'm not sure if it really works with 2.60 but it's not a disaster if it fails.